README.md

```diff
@@ -142,7 +142,9 @@ Supported options:
 | title | string   |  (optional) |
 | subject | string   | (optional) |
 | excludedActivityTypes | string   | (optional) |
-| showAppsToView | boolean | (optional) only android| 
+| failOnCancel | boolean | (defaults to true) On iOS, specifies whether promise should reject if user cancels share dialog (optional) |
+| showAppsToView | boolean | (optional) only android|
+>>>>>>> upstream/master
```

Comment:
remove this, please

Reply:
👍 not sure how I missed that before. It's removed now.

Reply:
thanks 🎉